root/inc/release-tools.html

```diff
@@ -35,8 +35,9 @@
     </button>
 </li>
 <li>
-    <form action="/search">
-        <input type="hidden" name="q" value="dist:<% release.distribution %>">
+    <form action="http://grep.metacpan.org/search">
```

Comment:
Should be https.

Reply:
updated to https, was not sure it was available :-)